src/components/NavBar.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { NavLink, useLocation } from 'react-router-dom';
+import Nav from 'react-bootstrap/Nav';
+import Stack from 'react-bootstrap/Stack';
+import Logo from './assets/planet_bigger.png';
+import NavStyles from './NavBar.module.css';
+
+const NavBar = () => {
+  const location = useLocation();
+  const rocketLinkStyle = (pathname) => {
+    if (pathname === '/rockets' || pathname === '/' || pathname === '') {
+      return `${NavStyles.linksStyles} ${NavStyles.mr1} ${NavStyles.active}`;
+    }
+    return `${NavStyles.linksStyles} ${NavStyles.mr1}`;
+  };
+
+  const missionsLinkStyle = (pathname) => {
+    if (pathname === '/mission') {
+      return `${NavStyles.linksStyles} ${NavStyles.mr1} ${NavStyles.active}`;
+    }
+    return `${NavStyles.linksStyles} ${NavStyles.mr1}`;
+  };
+
+  const dragonLinkStyle = (pathname) => {
+    if (pathname === '/dragon') {
+      return `${NavStyles.linksStyles} ${NavStyles.borderRight} ${NavStyles.active}`;
+    }
+    return `${NavStyles.linksStyles} ${NavStyles.borderRight}`;
+  };
+
+  const myProfileLinkStyle = (pathname) => {
+    if (pathname === '/myprofile') {
+      return `${NavStyles.linksStyles} ${NavStyles.ml1} ${NavStyles.active}`;
+    }
+    return `${NavStyles.linksStyles} ${NavStyles.ml1}`;
+  };
```

Comment:
Please refactor this code, you are repeating one function again and again, it is not according to the DRY principle. You can use loop for that.